v2/src/layer.ts

```diff
@@ -60,7 +60,7 @@ export function applyLayers(
         errors.push(errorMessage);
         return;
       }
-      lambdaLayerArn = getLambdaLayerArn(region, pythonLayerVersion, runtime, isARM, isNode, accountId);
+      lambdaLayerArn = getLambdaLayerArn(region, pythonLayerVersion, runtime, isARM, false, accountId);
```

Comment:
I noticed that there is a lot of repeated logic here. Can it be deduplicated?

Reply:
Yup, I wanna address it in another PR, too much to refactor if I do it here.